HW02-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,36 @@
+package homework;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class CustomerService {
+    TreeMap<Customer, String> customers = new TreeMap<>(Comparator.comparing(Customer::getScores));
```

Comment:
Поля private.
Тип лучше объявлять минимальным интерфейсом.
```suggestion
    private final NavigableMap<Customer, String> customers = new TreeMap<>(Comparator.comparing(Customer::getScores));
```